### PR TITLE
removed unused topics. Fixes RHOARDOC-1193

### PR DESCRIPTION
**Pull request by rhoads-zach**
@inoxx03 @tradej i need a sanity check on this. mind taking a look?

### CODE REVIEW

**Comment by Ladicek** [comment]
I abstain :-) Will verify during a diff check.

**Comment by tradej** [comment]
I have a sneaking suspicion some of the files are actually used, so I'll check this thoroughly tomorrow.

**Comment by rhoads-zach** [comment]
@inoxx03 @tradej fixed reaming ref to launcher script and updated contrib guide.

File: docs/topics/con_validating-guides.adoc [hunk 1]
```diff
@@ -39,3 +39,26 @@ To re-enable validation of the book, remove the `.ci-ignore` file, for example:
 $ rm docs/wf-swarm-runtime/.ci-ignore
 ----
 
+== Checking for Unused Topics
```
Comment (tradej) [inline docs/topics/con_validating-guides.adoc]: Zach, can you please move this into a separate module? This is not related to validation IMO.

**Comment by rhoads-zach** [comment]
@tradej amended PR with concept in separate topic
